README.md

```diff
@@ -192,7 +193,9 @@ const FOO = 1; // visibility missing!
 public const BAR = 2; // correct
 ```
 
-On PHP 7.0, this sniff must be manually excluded.
+Sniff provides the following settings:
+
+* `enabled` - explicitly enable or disable this sniff (by default, it will be only enabled when using PHP 7.1+)
```

Comment:
We want this option undocumented as said in https://github.com/slevomat/coding-standard/pull/135 :)